openeit/app.py

```diff
@@ -8,7 +8,7 @@
 
 import serial.tools.list_ports
 
-import OpenEIT.dashboard
+import openeit.dashboard
```

Comment:
I actually think OpenEIT is the better name than (openeit) (but am willing to change it, if the community at large prefers this). While this is in violation of PEP8 it is a conscious violation of PEP8 (which PEP8 explicitly supports).

My reasoning is that (a) it is in concordance with many other packages, e.g. OpenSSL, OpenGL, etc. of the form OpenTLA (b) openeit is hard to read and underscores are discouraged in package names.